src/mongodb.appup.src

```diff
@@ -1,27 +1,49 @@
 %% -*- mode: erlang -*-
-{"3.0.11",
+{"3.0.12",
  [
-  {"3.0.10", []},
+  {"3.0.11", [
+    {load_module, mc_topology, brutal_purge, soft_purge, []},
+    {load_module, mc_topology_logics, brutal_purge, soft_purge, []}
```

Comment:
For upgrading from 3.0.11 -> 3.0.12, we need to reload all the 4 modules that are changed using `load_module`, and add the new module `mc_util` using `{add_module, mc_util}`.